server/src/main/java/org/opensearch/cluster/action/index/NodeMappingRefreshAction.java

```diff
@@ -66,11 +72,17 @@ public class NodeMappingRefreshAction {
 
     private final TransportService transportService;
     private final MetadataMappingService metadataMappingService;
+    private final ThreadPool threadPool;
 
     @Inject
-    public NodeMappingRefreshAction(TransportService transportService, MetadataMappingService metadataMappingService) {
+    public NodeMappingRefreshAction(
```

Comment:
should have raised a different PR for these changes,

Reply:
Yes Shweta, I am going to remove this change to keep Data node side changes clean. Will add this changes into different PR where we will onboard refresh-mapping and shard-state actions into throttling framework.